### PR TITLE
fix: Reported design UI/UX issues with customizer-redesign

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Reported design UI/UX issues with customizer-redesign
- Task - https://trello.com/c/gay8HoEt/1201-final-zip-testing
- Section UI/UX
- - Replaced hard-coded default value with proper option in configs

### Types of changes
- Nonbreaking changes

### How has this been tested?
- With new customizer UI

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

File: inc/builder/type/base/assets/js/customizer-preview.js [hunk 1]
```diff
@@ -537,6 +537,37 @@ function astra_builder_widget_css( builder_type = 'header' ) {
 			selector + ' .widget-title'
 		);
 
+		astra_generate_outside_font_family_css(
```
Comment (vrundakansara) [inline inc/builder/type/base/assets/js/customizer-preview.js:540]: These options come from addon, shall we move them to the addon? @imnavanath

File: inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-layout-configs.php [hunk 2]
```diff
@@ -120,7 +120,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'type'        => 'control',
 					'control'     => 'ast-slider',
 					'section'     => 'woocommerce_product_catalog',
-					'default'     => 1200,
+					'default'     => astra_get_option( 'shop-archive-max-width' ),
```
Comment (vrundakansara) [inline inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-layout-configs.php:123]: Should we also add them from defaults?Ssince for new users no value will be set here.

File: inc/compatibility/edd/customizer/sections/layout/class-astra-edd-archive-layout-configs.php [hunk 3]
```diff
@@ -236,7 +236,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'type'        => 'control',
 					'control'     => 'ast-slider',
 					'section'     => 'section-edd-archive',
-					'default'     => 1200,
+					'default'     => astra_get_option( 'edd-archive-max-width' ),
```
Comment (vrundakansara) [inline inc/compatibility/edd/customizer/sections/layout/class-astra-edd-archive-layout-configs.php:239]: Should we also add them from defaults?Ssince for new users no value will be set here.

File: inc/customizer/configurations/layout/class-astra-blog-layout-configs.php [hunk 4]
```diff
@@ -73,7 +73,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'control'     => 'ast-slider',
 					'section'     => 'section-blog',
 					'transport'   => 'postMessage',
-					'default'     => 1200,
+					'default'     => astra_get_option( 'blog-max-width' ),
```
Comment (vrundakansara) [inline inc/customizer/configurations/layout/class-astra-blog-layout-configs.php:76]: Should we also add them from defaults?Ssince for new users no value will be set here.

File: inc/customizer/configurations/layout/class-astra-blog-single-layout-configs.php [hunk 5]
```diff
@@ -61,7 +61,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'control'     => 'ast-slider',
 					'section'     => 'section-blog-single',
 					'transport'   => 'postMessage',
-					'default'     => 1200,
+					'default'     => astra_get_option( 'blog-single-max-width' ),
```
Comment (vrundakansara) [inline inc/customizer/configurations/layout/class-astra-blog-single-layout-configs.php:64]: Should we also add them from defaults?Ssince for new users no value will be set here.

File: inc/customizer/configurations/layout/class-astra-site-layout-configs.php [hunk 6]
```diff
@@ -36,7 +36,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'name'        => ASTRA_THEME_SETTINGS . '[site-content-width]',
 					'type'        => 'control',
 					'control'     => 'ast-slider',
-					'default'     => 1200,
+					'default'     => astra_get_option( 'site-content-width' ),
```
Comment (vrundakansara) [inline inc/customizer/configurations/layout/class-astra-site-layout-configs.php:39]: Should we also add them from defaults?Ssince for new users no value will be set here.

**Review by vrundakansara — COMMENTED** [review]
@imnavanath - Please test these below options for new users here.

File: inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-layout-configs.php (hunk 2 above)
Comment (imnavanath) [reply inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-layout-configs.php:123]: All these replaced option already in defaults array, but no idea why it used from beginning. I've verified every option and then only I've replaced this with astra_get_option.

File: inc/compatibility/edd/customizer/sections/layout/class-astra-edd-archive-layout-configs.php (hunk 3 above)
Comment (imnavanath) [reply inc/compatibility/edd/customizer/sections/layout/class-astra-edd-archive-layout-configs.php:239]: All these replaced option already in defaults array, but no idea why it used from beginning. I've verified every option and then only I've replaced this with astra_get_option.

File: inc/customizer/configurations/layout/class-astra-blog-layout-configs.php (hunk 4 above)
Comment (imnavanath) [reply inc/customizer/configurations/layout/class-astra-blog-layout-configs.php:76]: All these replaced option already in defaults array, but no idea why it used from beginning. I've verified every option and then only I've replaced this with astra_get_option.

File: inc/customizer/configurations/layout/class-astra-blog-single-layout-configs.php (hunk 5 above)
Comment (imnavanath) [reply inc/customizer/configurations/layout/class-astra-blog-single-layout-configs.php:64]: All these replaced option already in defaults array, but no idea why it used from beginning. I've verified every option and then only I've replaced this with astra_get_option.

File: inc/customizer/configurations/layout/class-astra-site-layout-configs.php (hunk 6 above)
Comment (imnavanath) [reply inc/customizer/configurations/layout/class-astra-site-layout-configs.php:39]: All these replaced option already in defaults array, but no idea why it used from beginning. I've verified every option and then only I've replaced this with astra_get_option.

File: inc/builder/type/base/assets/js/customizer-preview.js (hunk 1 above)
Comment (imnavanath) [reply inc/builder/type/base/assets/js/customizer-preview.js]: Done 👍

**Review by rohitp-bsf — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**